Britannia/map_data/geographical_regions/geographical_region.txt

```diff
@@ -11,8 +11,41 @@
 #	These groups are mutually exclusive on the same tier & should cover every part of the map
 ###########################################################################
 
- 
-world_britannia_wales = {
+
+world_britannia_south = {
+	duchies = {
+		# Gwynedd
+		d_northgwynedd d_arfon d_osmaeliog d_meirion d_deceanglia d_westpowys
+		# Powys
+		d_deva d_luitcoyt d_eastpowys d_centpowys d_buelt d_fferyllwg d_magnis
+		# Lindum
+		d_lindum d_lerion
+		# Demetia
+		d_deheubarth d_ystrad_tywi d_brycheiniog
+		# Siluria
+		d_glywissing
+		# Gwent
+		d_gwent d_westgwent
+		# Glevum
+		d_centgwent d_corinium d_baddon
+		# Cynwidion
+		d_eastgwent d_calchfynydd
+		# Dumnonia
+		d_centdumnonia d_avalon d_westdumnonia
+		# Cornwall
+		d_penwith d_kernow
+		# Belgae
+		d_cunetio d_gradawc d_the_belgae d_eastdumnonia d_rhegin
+		# Cantuaria
+		d_cantuaria d_durobrivae
+		# Londinium
+		d_durobrivae d_lundein d_mincip d_trinovantia
+		# Icenia
+		d_grwath d_the_fens d_went
+	}
+}
+
+world_britannia_south_wales = {
```

Comment:
Should probably be renamed to custom_wales since we're not using it as a subregion for our big general world regions

Reply:
Fair, I had only renamed it to follow the vanilla region-subregion structure

Reply:
Yeah, I was going to suggest using it as a subregion (in which case the naming would make sense) but the split duchy makes that somewhat questionable.